addons/website_slides/static/src/components/activity/activity_tests.js

```diff
@@ -0,0 +1,124 @@
+odoo.define('website_slides/static/src/tests/activity_tests.js', function (require) {
+'use strict';
+
+const components = {
+    Activity: require('mail/static/src/components/activity/activity.js'),
+};
+
+const {
+    afterEach,
+    beforeEach,
+    createRootComponent,
+    start,
+} = require('mail/static/src/utils/test_utils.js');
+
+QUnit.module('website_slides', {}, function () {
+QUnit.module('components', {}, function () {
+QUnit.module('activity', {}, function () {
+QUnit.module('activity_tests.js', {
+    beforeEach() {
+        beforeEach(this);
+
+        this.createActivityComponent = async function (activity) {
+            await createRootComponent(this, components.Activity, {
+                props: { activityLocalId: activity.localId },
+                target: this.widget.el,
+            });
+        };
```

Comment:
Use arrow function `async activity =>`. `this.widget.el` relies on scope of `beforeEach`.

Reply:
Done!